thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/api/AnomalyFeedbackApi.java

```diff
@@ -60,4 +67,40 @@ public AnomalyFeedbackApi setComment(final String comment) {
     this.comment = comment;
     return this;
   }
+
+  public Date getCreated() {
+    return created;
+  }
+
+  public AnomalyFeedbackApi setCreated(Date created) {
+    this.created = created;
+    return this;
+  }
+
+  public Date getUpdated() {
+    return updated;
+  }
+
+  public AnomalyFeedbackApi setUpdated(Date updated) {
+    this.updated = updated;
+    return this;
+  }
+
+  public String getUpdatedBy() {
+    return updatedBy;
+  }
+
+  public AnomalyFeedbackApi setUpdatedBy(String updatedBy) {
+    this.updatedBy = updatedBy;
+    return this;
+  }
+
+  public List<AnomalyFeedbackReason> getReasons() {
```

Comment:
nitpick: we often use the word _cause_ in the backend code base and in user education, eg root _cause_ analysis. 
Maybe using _causes_ would be more clearer than _reasons_.

What do you think? 
The UI can always change if need be.

Reply:
Good suggestion. What about `AnomalyCause`?

Reply:
e.g. `AnomalyFeedbackReason` -> `AnomalyCause` and `getReasons` -> `getCauses`